packages/wrangler/src/dev/use-esbuild.ts

```diff
@@ -118,7 +142,7 @@ export function useEsbuild({
 						// First bundle, no need to update bundle
 						bundled = true;
 					} else {
-						updateBundle();
+						await updateBundle();
```

Comment:
Good catch! But are we now delaying showing build warnings in the if-block below? Should that if-block be moved up?